src/notebooks/IBMiSerializer.ts

```diff
@@ -0,0 +1,64 @@
+import * as vscode from 'vscode';
+import { TextDecoder, TextEncoder } from 'util';
+import { IBMiController } from './Controller';
+
+interface RawNotebookCell {
+  language: string;
+  value: string;
+  kind: vscode.NotebookCellKind;
+}
+
+export function notebookInit() {
+  const openNotebook = vscode.commands.registerCommand(`vscode-db2i.openNotebook`, (node) => {
+    const uri = node ? node.resourceUri : vscode.Uri.parse(`untitled:` + `notebook.inb`);
```

Comment:
Should there be a counter here that could be used to generate the file name?  As it is now, with the default file name `notebook.inb`, only one unsaved notebook can be open at a time.

Reply:
Fixed in a new commit. Changed the code to use a better VS Code API to open blank documents, which handles the title like this for us.